style.css

```diff
@@ -42,6 +42,7 @@ body {
 
 button {
   width: 250px;
+  cursor: pointer;
```

Comment:
we already have `cursor: pointer` in globals.css why do we need it here?

Reply:
This was not done by me. This change was already there.